test-fixtures/src/main/java/org/sonar/api/testfixtures/log/AbstractLogTester.java

```diff
@@ -51,11 +53,10 @@
   private final ConcurrentListAppender<ILoggingEvent> listAppender = new ConcurrentListAppender<>();
 
   protected void before() {
+    beforeFirstTest.set(false);
```

Comment:
Not sure I understand this.
`before` should be called before every test, assuming that this class is used as a `Rule` and not `ClassRule`.
If we want to preserve the default log value before this test ran, we should probably save it in the constructor and set it back to it in `after`.